core/libs/commonwealth/commonwealth/settings/tests/test_manager.py

```diff
@@ -0,0 +1,211 @@
+import os
+import pathlib
+import tempfile
+from typing import Any, Dict
+
+import pykson  # type: ignore
+
+from .. import manager, settings
+
+
+class SettingsV1(settings.BaseSettings):
+    VERSION = 1
+    version_1_variable = pykson.IntegerField(default_value=42)
+
+    def __init__(self, *args: str, **kwargs: int) -> None:
+        super().__init__(*args, **kwargs)
+
+        self.VERSION = SettingsV1.VERSION
```

Comment:
I'm almost sure that is unneeded. `self.VERSION` will already point to the `VERSION` class variable of the instantiated class.

Reply:
From my tests, `self.VERSION` is initialized from the base class `settings.BaseSettings` when the `super().__init__`  does the pykson initialization

Reply:
Humnnnn, right. Will take a look into that.